packages/manager/modules/sms/src/sms/batches/details/routing.js

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
+
+import component from './telecom-sms-batches-details.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service.batches.details', {
+    url: '/:batchId',
+    views: {
+      'smsView@sms.service': {
+        component: component.name,
+      },
+    },
+    params: {
+      previousState: '^',
+    },
+    resolve: {
+      batch: /* @ngInject */ ($transition$, batches) =>
+        batches.find(({ id }) => id === $transition$.params().batchId),
+      getOutgoingSms: /* @ngInject */ (iceberg, serviceName) => (batch) =>
+        iceberg(`/sms/${serviceName}/outgoing`)
+          .query()
+          .expand('CachedObjectList-Pages')
+          .execute({
+            'creationDatetime.from': batch.createdAt,
+            'creationDatetime.to': moment().format(),
+            batchId: batch.id,
+          })
+          .$promise.then(({ data }) => data),
+      getPttDetails: /* @ngInject */ ($http) => (outgoing) =>
+        $http({
+          url: '/sms/ptts',
+          method: 'GET',
+          params: {
+            ptt: outgoing.ptt,
+          },
+        }).then(({ data }) => data),
+      goBack: /* @ngInject */ ($state, $transition$) => (options) =>
+        $state.go($transition$.params().previousState, {}, options),
+      goToDelete: /* @ngInject */ ($state) => (outgoing) =>
+        $state.go('sms.service.batches.details.delete', {
+          outgoing,
+        }),
+      outgoingSms: /* @ngInject */ (batch, getOutgoingSms) =>
+        getOutgoingSms(batch),
+    },
+    translations: {
+      value: ['.', '..'],
```

Comment:
To be standalone, the module shouldn't need a parent's translations

Reply:
this is seem to be an outdated part of code, can you re-check please ? :)